src/libslic3r/Preset.cpp

```diff
@@ -877,7 +877,7 @@ static std::vector<std::string> s_Preset_printer_options {
     "best_object_pos","head_wrap_detect_zone",
     //SoftFever
     "host_type", "print_host", "printhost_apikey", "bbl_use_printhost",
-    "print_host_webui",
+    "print_host_webui", "bbl_use_print_host_webui", "simplyprint_post_upload_action",
```

Comment:
Can we make it generic e.g. call it `bbl_post_upload_action`?
So in the future, other print host services can use this parameter too?

Reply:
Done

Reply:
Damn... Forgot there is an option with this same name already. I'll revert.
This option is specific to SimplyPrint only, I don't think there is the need to make it generic.

Reply:
Hehe, I didn't notice that as well.
Looks like we can use that mechanism.

Reply:
> Damn... Forgot there is an enum with this same name already. I'll revert. This option is specific to SimplyPrint only, I don't think there is the need to make it generic.

Can you elaborate why it's can't be used for other third-party host service? Technically, it applies to all other cloud service provider.
Hence, `post_upload_action` would be better.